src/Browser.ts

```diff
@@ -199,6 +213,7 @@ class Browser {
     public async getProfile(userId: number): Promise<IProfileData> {
         const $ = await this._get$(`users/${userId}`);
 
+        const id = userId;
```

Comment:
If you're already passing userId to the function, why bother returning it? You already know it if you're calling this function.

Reply:
I am also trying to store the account's profile and reuse it instead of having to create another getProfile function call.

    // Store my profile outside of the message event
    const _me = await client.getMe();
    const me = await client._browser.getProfile(_me.id);
    me.id = _me.id; // <-- current hack, because getProfile() doesn't return id

This will allow me to later use it interchangeably like this:

    // Get details of user who triggered the message
    const user = msg.userId == me.id ? me : await client._browser.getProfile(msg.userId);

    console.log(user.id); // <-- able to use user.id whether message is from self or others